mammoth/docx/body_xml.py

```diff
@@ -350,30 +349,49 @@ def break_(element):
             warning = results.warning("Unsupported break type: {0}".format(break_type))
             return _empty_result_with_message(warning)
 
-    
     def inline(element):
         properties = element.find_child_or_null("wp:docPr").attributes
+
         if properties.get("descr", "").strip():
             alt_text = properties.get("descr")
         else:
             alt_text = properties.get("title")
+
+        exts = element.find_child_or_null("a:graphic") \
+            .find_child_or_null("a:graphicData") \
+            .find_child_or_null("pic:pic") \
+            .find_child_or_null("pic:spPr") \
+            .find_child_or_null("a:xfrm") \
+            .find_child_or_null("a:ext")
+
+        if exts.attributes:
```

Comment:
I don't think you need this check, the true branch will handle both cases since `attributes` is always a dictionary.

Reply:
It was to prevent a TypeError:

```
try: width  = int( exts.attributes.get("cx") )
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Said that, I have added the `TypeError` to the exception clause where it belongs 👍